miqa/core/tasks.py

```diff
@@ -49,6 +49,21 @@ def _download_from_s3(path: str, public: bool) -> bytes:
     return buf.getvalue()
 
 
+@shared_task
+def reset_demo():
+    if not settings.DEMO_MODE:
+        raise Exception('Demo Mode is false; noop.')
+    try:
+        demo_project = Project.objects.get(name='Demo Project')
+    except Project.DoesNotExist:
+        raise Exception('No Demo Project; noop.')
```

Comment:
```suggestion
    demo_project = Project.objects.get(name='Demo Project')
```

I'm not clear on the value of transforming the exception type and message. This exception is for the benefit of other developers and `Project.DoesNotExist`, raised from this point in the code or just within the `reset_demo` function, seems pretty obvious already. Maybe add a code comment if you really don't think it's apparent why the code expects that exactly one demo Project should exist.

Reply:
Removed in [418868c](https://github.com/OpenImaging/miqa/pull/635/commits/418868c2527eca6e3bbd4f287477a48908834cce)